resolwe/flow/utils/__init__.py

```diff
@@ -11,3 +11,14 @@
 
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
+
+import hashlib
+
+
+def get_data_checksum(proc_input, proc_slug, proc_version):
+    """Compute checksum of processor inputs, name and version."""
+    hash_ = hashlib.sha1()
```

Comment:
It's a built-in function and pylint is not happy with overriding it. I've renamed it to `checksum`